Signal/src/Models/TSMessageAdapaters/TSMessageAdapter.m

```diff
@@ -354,11 +353,9 @@ - (NSString *)text {
 
 - (NSUInteger)messageHash
 {
-    if (self.isMediaMessage) {
-        return [self.mediaItem mediaHash];
```

Comment:
Is this going to properly invalidate the bubble size when a media message changes state?

e.g. when it goes from from "downloading" -> "downloaded"?